SpywareFilter/sections/whitelist.txt

```diff
@@ -8,6 +8,8 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60961
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
 @@||secure.quantserve.com/quant.js$domain=fruitlab.com
+! https://github.com/AdguardTeam/AdguardFilters/pull/61240
+@@||googletagmanager.com/gtag/js$domain=klix.ba
```

Comment:
for fixing the slider, for example here
https://www.klix.ba/magazin/showbiz/katie-price-slomila-oba-stopala-krece-se-puzuci-na-koljenima/200810046